### PR TITLE
all: cleanup unused code, part 2

### PR DESCRIPTION
**Pull request by awly**
And enable U1000 check in staticcheck.

Updates #cleanup

### CODE REVIEW

**Comment by awly** [comment]
Shucks, looks like I forgot to run under different GOOSes

File: control/controlbase/noiseexplorer_test.go [hunk 1]
```diff
@@ -471,5 +441,3 @@ func RecvMessage(session *noisesession, message *messagebuffer) (*noisesession,
 	session.mc = session.mc + 1
 	return session, plaintext, valid
 }
-
-func main() {}
```
Comment (creachadair) [inline control/controlbase/noiseexplorer_test.go:475]: This is really weird, I'm having trouble understanding why this was ever necessary.

**Review by creachadair — APPROVED** [review]

**Comment by creachadair** [comment]
Hmm, it looks like there are a few more unused bits in the tags.

**Comment by awly** [comment]
Added all the windows/darwin findings. @creachadair mind skimming the diff one more time? :pray:

**Review by creachadair — APPROVED** [review]

**Comment by dblohm7** [comment]
The next time we run one of these, can we PLEASE tag in more reviewers?

**Pull request merged**